content/docs/api/json-rpc.mdx

```diff
@@ -0,0 +1,761 @@
+---
+title: JSON-RPC API
+description: JSON-RPC API
+---
+
+The Shardeum JSON-RPC API enables developers to interact directly with the Shardeum network. This documentation provides a quick reference to essential methods for querying network data, managing transactions, and more.
+
+## web3_clientVersion
+
+Returns the current client version.
+
+**Parameters**
+
+None
+
+**Returns**
+
+`result` - The current client version
+
+**Example**
+
+```bash
+curl https://your-rpc-endpoint \
+  -X POST \
+  -H "Content-Type: application/json" \
+  --data '{"method":"web3_clientVersion","id":1,"jsonrpc":"2.0"}'
+```
+
+## web3_sha3
+
+Returns Keccak-256 (_not_ the standardized SHA3-256) of the given data.
+
+**Parameters**
+
+data - The data in hexadecimal form to convert into a SHA3 hash
+
+**Returns**
+
+`result` - The SHA3 result of the given string.
+
+**Example**
+
+```bash
+curl https://your-rpc-endpoint \
+  -X POST \
+  -H "Content-Type: application/json" \
+  --data '{"jsonrpc":"2.0","method":"web3_sha3","params":["data"],"id":1}'
+```
+
+## net_version
+
+Returns the current network id.
+
+**Parameters**
+
+None
+
+**Returns**
+
+`result` - The current network id.
+
+The full list of current network IDs is available at [chainlist.org](https://chainlist.org). Some common ones are:
+
+- `1`: Ethereum Mainnet
+- `5`: Goerli testnet
+- `11155111`: Sepolia testnet
+
+**Example**
+
+```bash
+curl https://your-rpc-endpoint \
+  -X POST \
+  -H "Content-Type: application/json" \
+  --data '{"method":"net_version","id":1,"jsonrpc":"2.0"}'
+```
+
+## net_listening
+
+Returns `true` if client is actively listening for network connections.
+
+**Parameters**
+
+None
+
+**Returns**
+
+`result` - `true` when listening, otherwise `false`.
+
+**Example**
+
+```bash
+curl https://your-rpc-endpoint \
+  -X POST \
+  -H "Content-Type: application/json" \
+  --data '{"method":"net_listening","id":1,"jsonrpc":"2.0"}'
+```
+
+## net_peerCount
```

Comment:
Is this relevant in Shardeum?

Reply:
not sure, i have asked mehdi to double check, as of now, i am removing it

Reply:
yes, it's unsupported